frontend/packages/dev-console/src/components/monitoring/metrics/__tests__/MonitoringMetrics.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import MonitoringMetrics from '../MonitoringMetrics';
+
+describe('Monitoring Metrics Tab', () => {
+  it('should render Monitoring Metrics tab', () => {
+    const wrapper = shallow(<MonitoringMetrics />);
+    expect(wrapper.contains(<title>Metrics</title>)).toBe(true);
```

Comment:
Same here, I'd drop this file too. They can be created later to handle any functionality that is added.